src/views/Inventory.vue

```diff
@@ -33,6 +33,26 @@
               </ion-label>
             </ion-item>
           </ion-card>
+
+          <ion-card>
```

Comment:
I think we should create a separate component and reuse throughout the app

Reply:
Yes sir. We do have a separate component called "MoreJobs" but I created it in the PR I've added functionality part.
PR - https://github.com/hotwax/job-manager/pull/250
Do you want me to display static data through a separate component "MoreJobs"?